pyatlas_checks/pyatlas_checks/cli/cli.py

```diff
@@ -0,0 +1,20 @@
+import click
+from pyatlas_checks.commands.test_command import TestCommand
+
+
+@click.group()
+@click.option('--verbose', is_flag=True)
+@click.pass_context
+def cli(ctx, verbose):
+    ctx.obj = {
+        'verbose': verbose
+    }
+
+
+@cli.command()
+@click.argument('test_string')
+@click.pass_context
+def test_command(ctx, test_string):
```

Comment:
Do you have to declare a name here? How would you call this from the cli?

Reply:
@danielduhh To call this command, all you have to do is pass:
```
pyatlas test-command
```
Of course, calling the above will throw an error since we didn't pass the required parameter.
Click automatically converts all underscores in method names to dashes.

Reply:
💯